<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>496</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>329</width>
    <height>496</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="GroupAll">
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupAuthorization">
        <property name="title">
         <string>Authentication</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lb_login">
           <property name="text">
            <string>Login:</string>
           </property>
           <property name="buddy">
            <cstring>le_login</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_login"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lb_password">
           <property name="text">
            <string>Password:</string>
           </property>
           <property name="buddy">
            <cstring>le_password</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_password"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="groupHeader">
        <item row="2" column="0">
         <spacer name="vs_spaceUpper">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lb_header">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>Connection to database</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="Line" name="l_lineUpper">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupParameters">
        <property name="title">
         <string>Connection Parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="2">
          <widget class="QLabel" name="lb_port">
           <property name="text">
            <string>Port:</string>
           </property>
           <property name="buddy">
            <cstring>sb_port</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="sb_port">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_hostName"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lb_hostName">
           <property name="text">
            <string>Host name:</string>
           </property>
           <property name="buddy">
            <cstring>le_hostName</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lb_dbName">
           <property name="text">
            <string>Database name:</string>
           </property>
           <property name="buddy">
            <cstring>le_dbName</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="le_dbName"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <layout class="QGridLayout" name="groupAction">
        <item row="0" column="0">
         <widget class="QCheckBox" name="cb_connect">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pb_ok">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pb_close">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QGridLayout" name="groupAbout">
        <item row="0" column="0">
         <widget class="Line" name="l_lineBottom">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="pb_about">
          <property name="text">
           <string>About</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="vs_spaceBottom">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>329</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>le_dbName</tabstop>
  <tabstop>le_hostName</tabstop>
  <tabstop>sb_port</tabstop>
  <tabstop>le_login</tabstop>
  <tabstop>le_password</tabstop>
  <tabstop>cb_connect</tabstop>
  <tabstop>pb_ok</tabstop>
  <tabstop>pb_close</tabstop>
  <tabstop>pb_about</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pb_close</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
